src/unistd.rs

```diff
@@ -2778,3 +2778,42 @@ impl Group {
         })
     }
 }
+
+/// Get the name of the terminal device that is open on file descriptor fd
+/// (see [`ttyname(3)`](http://man7.org/linux/man-pages/man3/ttyname.3.html)).
+///
+/// # Safety
+///
+/// `ttyname()` mutates global variables and is *not* threadsafe.
+/// Mutating global variables is always considered `unsafe` by Rust and this
+/// function is marked as `unsafe` to reflect that.
+///
+/// For a threadsafe and non-`unsafe` alternative, see `ttyname_r()`.
+#[inline]
```

Comment:
Don't use `#[inline]` for new code.  The compiler can figure it out, at least when LTO is enabled.

Reply:
:+1: